extension.js

```diff
@@ -19,8 +19,8 @@ import * as Extension from 'resource:///org/gnome/shell/extensions/extension.js'
 import * as StatusNotifierWatcher from './statusNotifierWatcher.js';
 import * as Interfaces from './interfaces.js';
 import * as TrayIconsManager from './trayIconsManager.js';
+import * as SettingsManager from './settingsManager.js';
 import * as Util from './util.js';
-import {SettingsManager} from './settingsManager.js';
```

Comment:
This change is unrelated though, also it may not work as otherwise `SettingsManager.SettingsManager` should be used instead.

Reply:
Ooops was a leftover for uniformization, will remove it